ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/cards/LoiCardUtil.kt

```diff
@@ -43,6 +45,23 @@ object LoiCardUtil {
     }
   }
 
+  /**
+   * Converts float/double IDs into ints if the decimal portion is 0; ignores all other id types.
+   * * ex. "1234.00" -> "1234",
+   * * "1234.01" -> "1234.01",
+   * * "abcdefg" -> "abcdefg",
+   */
+  private fun String.formatAsId(): String {
+    return when {
+      toDoubleOrNull() != null && toDouble() - floor(toDouble()) == 0.0 -> {
+        String.format("%.0f", toDouble())
```

Comment:
I don't think we need this; IDs will rarely, if ever, have decimals, and will rarely be numeric. I think we carried forward numeric IDs from GeoJSON in case users want them to also be numbers in the exported CSV. Please feel free to just convert to a string if the id is a number. Thanks!

Reply:
Going to drop this PR in that case! It sounds like the original issue was a one off with unexpected inputs.